pkg/clustermesh/endpointslicesync/cell.go

```diff
@@ -7,9 +7,11 @@ import (
 	"time"
 
 	"github.com/cilium/hive/cell"
+	"github.com/cilium/hive/job"
```

Comment:
IIUC, you need to import `"github.com/cilium/cilium/hive"`. Otherwise, the Job will not be registered to the lifecycle. I might be wrong. There were a bunch of changes recently. Did the current implementation work?